admin_manual/configuration_server/caching_configuration.rst

```diff
@@ -207,3 +211,40 @@ Cache Directory Location
 The cache directory defaults to ``data/$user/cache`` where ``$user`` is the 
 current user. You may use the ``'cache_path'`` directive in ``config.php``
 (See :doc:`config_sample_php_parameters`) to select a different location.
+
+Recommendation Based on Type of Deployment
+------------------------------------------
+
+Small/private home server
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Only use APCu::
+
+    'memcache.local' => '\OC\Memcache\APCu',
+
+Small organization, single-server setup
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Use APCu for local caching, Redis for file locking::
+
+ 'memcache.local' => '\OC\Memcache\APCu',
+ 'memcache.locking' => '\OC\Memcache\Redis',
+  'redis' => array(
+       'host' => 'YOUR_CHOICE',
```

Comment:
This is good stuff. One suggestion: replace YOUR_CHOICE with the same examples used elsewhere in the doc.